AEPMessaging/Sources/Messaging.swift

```diff
@@ -760,30 +766,45 @@ public class Messaging: NSObject, Extension {
         }
     }
 
-    /// Handles Rules Consequence events containing schemas
-    private func handleRulesResponse(_ event: Event) {
-        guard event.isSchemaConsequence, event.data != nil else {
-            Log.trace(label: MessagingConstants.LOG_TAG, "Ignoring rule consequence event. Either consequence is not of type 'schema' or 'eventData' is nil.")
+    /// Handles rules engine consequences
+    private func handleRulesConsequence(_ event: Event) {
+        guard event.data != nil else {
```

Comment:
why did we remove the check for `isSchemaConsequence`?  this is the fastest way to filter out that we don't need to handle events triggered by the rules engine